src/pages/home/base/mobile-core/lifecycle/configuration-keys.md

```diff
@@ -16,7 +16,7 @@ keywords:
 
 This configuration setting is only used in the Analytics use case, when using the [Lifecycle data content response](./event-reference.md#lifecycle-data-content-response) event to determine session length. <br/><br/> In the Platform use case, events are dispatched based on [Lifecycle Application Foreground](./event-reference.md#lifecycle-application-foreground) and [Lifecycle Application Background](./event-reference.md#lifecycle-application-background) and not on a set session timeout.
 
-Specifies the length of time, in seconds, that must elapse between the time the app is launched and before the launch is considered to be a new session. This timeout also applies when your application is sent to the background and reactivated.
+Specifies the length of time, in seconds, that must elapse between the time the app is sent to the background and before the next launch is considered to be a new session.
```

Comment:
new wording isn't quite flowing for me - it's a lot to try and communicate in one sentence 😄 

`Number of seconds that must elapse between the app entering the background and subsequently re-entering the foreground in order for the SDK to create a new session.`

?